.circleci/config.yml

```diff
@@ -15,26 +15,6 @@ e2eTest: &e2eTest
 
     - run: ./e2e-harness setup --remote=false
 
-    - run:
```

Comment:
Better not removing this pls, we are going to need the CNR server in the full e2e test.

Reply:
It is not used right now so I dropped it. We can add back when needed.